python/paddle/fluid/layers/nn.py

```diff
@@ -975,6 +976,44 @@ def cross_entropy(input, label, soft_label=False):
     return out
 
 
+def log_loss(input, label, epsilon=1e-4):
+    """
+    **Negative Log Loss Layer**
+
+    This layer accepts input predictions and target label and returns the
+    negative log loss.
+
+    .. math::
+
+        Out = -label * log(input + epsilon) - (1 - label) * log(1 - input + epsilon)
+
+    Args:
+        input (Variable|list):  a 2-D tensor with shape [N x 1], where N is the batch size.
```

Comment:
exceeded 80 columns

Reply:
sorry... in my memory, the limitation of google style is 100. I will update soon

Reply:
@pkuyym done

Reply:
problem fixed